src/dtl/dyad_flux_log.h

```diff
@@ -17,11 +21,14 @@
     do {                            \
     } while (0)
 #else
-#define DYAD_LOG_INFO(dyad_ctx, ...) \
-    flux_log (dyad_ctx->h, LOG_INFO, __VA_ARGS__)
+#define DYAD_LOG_INFO(dyad_ctx, ...) flux_log (dyad_ctx->h, LOG_INFO, __VA_ARGS__)
```

Comment:
space

Reply:
Since there are not spaces after function-like macros in `flux-core`, I'd argue that we shouldn't add spaces in DYAD either.